deploy/scripts/src/mobile/sts.ts

```diff
@@ -203,6 +224,26 @@ const reauthenticationContextData: ReauthenticationContext[] = new SharedArray('
   }
 })
 
+const exchangeRefreshTokenContextData: ExchangeRefreshTokenContext[] = new SharedArray(
+  'exchangeRefreshTokenContext',
+  () => {
+    const exchangeRefreshTokenDataFile = `./data/sts-refresh-token-test-data-${getEnv('ENVIRONMENT')}.csv`
+    try {
+      const exchangeRefreshTokenContextData = open(exchangeRefreshTokenDataFile)
+      const parsedCsvData = exchangeRefreshTokenContextData
+        .split('\n')
+        .filter(line => line.trim() !== '') // Remove empty lines
+        .map(line => ({ refreshToken: line.trim() }))
+      return parsedCsvData
```

Comment:
Suggestion for readability, I'm not sure if `parsedCsvData` is particularly clear here:
```suggestion
      return exchangeRefreshTokenContextData
        .split('\n')
        .filter(line => line.trim() !== '') // Remove empty lines
        .map(line => ({ refreshToken: line.trim() }))
```